test-lib.sh

```diff
@@ -894,6 +894,26 @@ ct_s2i_build_as_df()
     local src_image=$1; shift
     local dst_image=$1; shift
     local s2i_args="$*";
+
+    ct_s2i_build_as_df_build_args "$app_path" "$src_image" "$dst_image" "" "$s2i_args"
+}
+
+# ct_s2i_build_as_df_build_args APP_PATH SRC_IMAGE DST_IMAGE [S2I_ARGS]
```

Comment:
```
-  # ct_s2i_build_as_df_build_args APP_PATH SRC_IMAGE DST_IMAGE [S2I_ARGS]
+  # ct_s2i_build_as_df_build_args APP_PATH SRC_IMAGE DST_IMAGE BUILD_ARGS [S2I_ARGS]
```

Just a nitpick in comment.

Also could you please explain in comment of `ct_s2i_build_as_df` function why we created the wrapper?

Reply:
It should be explained by https://github.com/sclorg/container-common-scripts/pull/368/commits/ca4f716c7b500c2ad9796a7c79b94f42e9848474